show-coordinates-in-multiple-formats/src/main/java/com/esri/arcgismaps/sample/showcoordinatesinmultipleformats/screens/MainScreen.kt

```diff
@@ -49,9 +48,7 @@ fun MainScreen(sampleName: String) {
     // graphics overlay for the MapView to draw the graphics
     val graphicsOverlay = remember { GraphicsOverlay() }
     // the collection of graphics overlays used by the MapView
-    val graphicsOverlayCollection = rememberGraphicsOverlayCollection().apply {
-        add(graphicsOverlay)
-    }
+    val graphicsOverlays = remember { listOf(graphicsOverlay) }
 
     graphicsOverlay.graphics.add(mapViewModel.coordinateLocationGraphic)
```

Comment:
The list `graphicsOverlays` would need to be remembered, but it is not needed for the graphics overlay object:
```kotlin
    // graphics overlay to display a graphics of the coordinate location
    val graphicsOverlay = GraphicsOverlay().apply {
        graphics.add(mapViewModel.coordinateLocationGraphic)    
    }
    // the collection of graphics overlays used by the MapView
    val graphicsOverlays = remember { listOf(graphicsOverlay) }
```

We follow this approach in the microapps ([SceneAnalysisOverlayApp](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/blob/a12e5447bae37665aefec929f0e424725a9e20f7/microapps/SceneViewAnalysisOverlayApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewanalysisoverlayapp/screens/MainScreen.kt#L97), [MapViewGeometryEditorApp](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/blob/a12e5447bae37665aefec929f0e424725a9e20f7/microapps/MapViewGeometryEditorApp/app/src/main/java/com/arcgismaps/toolkit/mapviewgeometryeditorapp/screens/MainScreen.kt#L86))